pandas/tests/extension/numpy_/test_numpy_nested.py

```diff
@@ -0,0 +1,276 @@
+import pytest
+
+import pandas as pd
```

Comment:
can you add a comment here on what this particular file is testing. (as a casual glance makes it look very similar to test_numpy.py)

Reply:
@TomAugspurger but still this does not answer the question of why you duplicated things

Reply:
does https://github.com/pandas-dev/pandas/pull/24993/files#diff-c77963c2757e522c9ed516402633c932R6 make sense?

Reply:
it makes sense, I suppose enough. Problem is a future reader may not understand exactly what you are getting at here.

Reply:
> @TomAugspurger why don't you just add a marker to specific tests? (and skip on that)

> I'm not aware of a way for a test marker to get access to the value of another fixture.

@TomAugspurger I fear that I may have led you in the wrong direction with the first example on just including the fixture that you want the value of in the function signature. that was just to show that fixtures can be added to the function signature and that additional unwanted permutations would not occur.

from the pytest docs on request.getfixturevalue.. "Declaring fixtures via function argument is recommended where possible. But if you can only decide whether to use another fixture at test setup time, you may use this function to retrieve it inside a fixture or test function body."

hence why i mentioned the function argument approach first.

@jreback is right about adding markers, and the `pytest.mark.usefixtures` is probably the appropriate marker to use.

if this marker was used only on tests that depended on the `dtype` fixture, then the autouse fixture I suggested could be used without the `request.getfixturevalue('dtype')` and `dtype` included in the fixture signature along with the `request` fixture which gives access to the class, instance and function to determine if the test should be skipped.

> Good to merge?

My comments should not be taken as a reason not to merge. A follow-on PR could look into this approach.

Reply:
I'm relatively happy with current approach. It's lower-tech which is fine for me in tests.